webpack/components/OpenscapRemediationWizard/steps/ReviewRemediation.js

```diff
@@ -0,0 +1,186 @@
+        );
+
+  const rebootAlertTitle = isRebootRequired()
+    ? __('A reboot is required.')
+    : __('A reboot might be needed.');
+
+  const actions = (
+    <React.Fragment>
+      <CodeBlockAction>
+        <ClipboardCopyButton
+          id="basic-copy-button"
+          textId="code-content"
+          aria-label="Copy to clipboard"
+          onClick={e => onCopyClick(e, snippetText)}
+          exitDelay={copied ? 1500 : 600}
+          maxWidth="110px"
+          variant="plain"
+          onTooltipHidden={() => setCopied(false)}
+        >
+          {copied
+            ? __('Successfully copied to clipboard!')
+            : __('Copy to clipboard')}
+        </ClipboardCopyButton>
+      </CodeBlockAction>
+    </React.Fragment>
+  );
+
+  return (
+    <>
+      <WizardHeader
+        title={__('Review remediation')}
+        description={description}
+      />
+      <Grid hasGutter>
+        <br />
+        <GridItem>
+          <Alert
+            ouiaId="review-alert"
+            variant="danger"
+            title={`${__(
+              'Do not implement any of the recommended remedial actions or scripts without first testing them in a non-production environment.'
+            )}
+              ${__('Remediation might render the system non-functional.')}`}
+          />
+        </GridItem>
+        <GridItem md={12} span={4} rowSpan={1}>
+          <ViewSelectedHostsLink
+            isAllHostsSelected={isAllHostsSelected}
+            hostIdsParam={hostIdsParam}
+            defaultFailedHostsSearch={defaultFailedHostsSearch}
+          />
+        </GridItem>
+        <GridItem md={12} span={4} rowSpan={1}>
+          <Button
+            variant="link"
+            icon={<ExternalLinkSquareAltIcon />}
+            iconPosition="right"
+            target="_blank"
+            component="a"
+            href={foremanUrl(`${getHostsPageUrl(true)}/${hostName}`)}
+          >
+            {hostName}
+          </Button>{' '}
+        </GridItem>
+        <GridItem md={12} span={8} rowSpan={1}>
+          <Button
+            variant="link"
+            icon={<ExternalLinkSquareAltIcon />}
+            iconPosition="right"
+            target="_blank"
+            component="a"
+            href={foremanUrl(
+              `${HOSTS_PATH}/?search=${FAIL_RULE_SEARCH} = ${source}`
+            )}
+          >
+            {__('Other hosts failing this rule')}
+          </Button>
+        </GridItem>
+        {checkForReboot() ? (
+          <>
+            <GridItem span={12} rowSpan={1}>
+              <Alert
+                ouiaId="reboot-alert"
+                variant={isRebootRequired() ? 'warning' : 'info'}
+                title={rebootAlertTitle}
+              >
+                {method === 'manual' ? null : (
+                  <p>
+                    {__(
+                      'You can tick the checkbox below to reboot the system(s) automatically after the remediation is applied.'
```

Comment:
You probably don't need this. The checkbox that says "Reboot the system(s)" is self-explanatory.

Removing this line from the info box might actually help because this part of the wizard is a bit too complex already; simplifying it by dropping the line would help.